_data/program_2021.yml

```diff
@@ -0,0 +1,48 @@
+- date: "2021-03-24"
+  dateReadable: "Day 1: Wednesday, 24<sup>th</sup> March 2021"
+  tracks:
+    - { title: "TrainTrack", color: "#90be4e" }
+    - { title: "HackTrack", color: "#03a9f4" }
+  timeSlots:
+    - {
+        startTime: "08:00",
+        endTime: "09:00",
+        sessionTitle: "coffee break",
+        speaker,
+        sessionType,
+        slides,
+      }
+    - {
+        startTime: "09:00",
+        endTime: "10:00",
+        sessionTitle: "Best session event",
+        speaker: "Marie Curie",
+        sessionType,
+        slides: "google.com",
+      }
+
+- date: "2021-03-25"
+  dateReadable: "Day 2: 25th March 2021"
+  tracks:
+    - { title: "TrainTrack", color: "#90be4e" }
+    - { title: "HackTrack", color: "#03a9f4" }
+  timeSlots:
+    - { startTime: "08:00", endTime: "09:00", sessionTitle: "coffee break" }
+    - {
+        startTime: "09:00",
+        endTime: "10:00",
+        sessionTitle: "Best session event",
+      }
+
+- date: "2021-03-26"
+  dateReadable: "Day 2: 26th March 2021"
+  tracks:
```

Comment:
I don't see different columns for each track in the current (empty) program. Is that because `program-table.html` does not yet contain anything related to `tracks`?

Reply:
yup exactly.
my thought was that we could add this later